praw/models/user.py

```diff
@@ -1,22 +1,57 @@
 """Provides the User class."""
-from ..const import API_PATH
 from .base import PRAWBase
 from .listing.generator import ListingGenerator
 from .reddit.redditor import Redditor
 from .reddit.subreddit import Subreddit
+from ..const import API_PATH
```

Comment:
Is there a reason this line was moved? I think the project consistently imports the more top-level models first before the deeper ones.

Reply:
PyCharm likes to order relative imports in the same directory above relative imports in a parent directory. I'll revert this change.